src/frontend/src/pages/LoginPage/Login.tsx

```diff
@@ -75,7 +75,7 @@ const Login = () => {
     redirectAfterLogin();
   };
 
-  const handleFailure = (response: any) => {
+  const handleFailure = (response: { children: any }) => {
```

Comment:
you still have a use of any here

Reply:
I pinged u on this ticket on slack, I don't think changing type on this one is doable (correct me if I'm wrong)